### DISCUSSION TITLE
Some questions regarding the NGP adapter

### DISCUSSION THREAD

**Discussion by solleo1989** [sanni/cartreader#856] [Need help]
### OSCR firmware version

HW5 Rev5

### OSCR hardware version

HW5 Rev5

### System used

NeoGeo Pocket (Color)

### Describe the issue.

[My setup]
HW5 Rev5 + Retail version Neo Geo Pocket pcb and soldered a OEM NGP cartridge slot. 
![image](https://github.com/sanni/cartreader/assets/5420867/74e89498-a9e4-4f31-877c-b03dfdf89b6c)
![image](https://github.com/sanni/cartreader/assets/5420867/b7c8e258-9dcc-4d26-9122-a6d141b70869)

I have verified each pin, it seemed there's no bridge or disconnecting issue.
But when I tried to dump the cartridge, it showed a "cartridge read error".
![image](https://github.com/sanni/cartreader/assets/5420867/1df03f59-6844-44a4-a6c4-c71241c1bbd5)

I have also tried different NGP/C carts, none of them is working. 
I also noticed [there are some ppl succeeded with SSCI-034173 cart slot](https://github.com/sanni/cartreader/discussions/614#discussioncomment-4209432). 
I'm wondering what's the difference ? Or maybe my soldering issue ?

Any help? 
Really appreciate !

**Comment by PsyK0p4T** [comment]
Heya
I'm using a NGP cart slot too here, and already dumped this game various times so... it should be ok ;)

Random things you can check : 
- no shortcut between pins
- conductivity is ok between your cartridge slot, and the adapter bottom pins
- your snes slot is working well if you try to dump other cartridges (snes ones, or different consoles using adapters)
- you're using 3V for NGP
- open the cartridge and clean the pcb pins very well (a white rubber gives good results) or q-tip with alcohol if you have no tool to open it. Most read errors are coming from here. You can also clean the adapter pins if needed.

**Reply by solleo1989** [reply]
Quick Q: just curious, are you using HW5 or HW4/3? I tested on my HW3, NGP adapter works fine, but not my HW5. I'll check the SNES slot tmr. Thanks for your answer, really appreciate!

**Reply by solleo1989** [reply]
HW5 I'm referring to the auto voltage switch version. :p

**Reply by PsyK0p4T** [reply]
I'm using a hw5 with manual voltage selection

**Reply by solleo1989** [reply]
Thanks for you help! 
I think it should be my 6-slots pcb or SNES cart slot issue. I just soldered a new 6 slots board, and it's working now, even though the SNES cart slot looks fine to me. :p

**Reply by PsyK0p4T** [reply]
hu.. ok.
Did the snes slot work correctly for other systems ? Like SNES carts, or other ones with adapters ?

**Reply by solleo1989** [reply]
The faulty one also has issue on SNES cart dump, even though I do remeber the first time it was working. Anyway, now it working for both SNES and NGP/WS carts now.
